### DISCUSSION TITLE
Not hearing my keyboard on headphones

### DISCUSSION THREAD

**Discussion by jedlevine** [jamulussoftware/jamulus#1538] [General]
New jamulus user. Connected with friends through a PreSonus interface. Hear them, they hear me on keyboard, but I don't hear my keyboard in the mix on my headphones.

**Comment by ann0see** [comment]
Hi @jedlevine did you check if your audio output works (headphone plugged in, output gain high enough, especially: **is local monitoring disabled**, system volume high enough): https://jamulus.io/wiki/Client-Troubleshooting#dont-hear-any-soundothers-dont-hear-you ?

Also: Do you use ASIO4ALL (I don't think this is related)? If you do, please avoid it and use a native ASIO driver if possible. If you really **need** ASIO4ALL (usually you don't if you own an interface) https://jamulus.io/wiki/Installation-for-Windows#setting-up-asio4all might help.

**Reply by jedlevine** [reply]
My interface is a PreSonus Studio 24C. There was only one PreSonus option in the jamulus drop-down menu. When I tried that one, it wasn't picking up my interface at all, so I then tried ASIO4ALL.

**Reply by ann0see** [reply]
I'd try to remove ASIO4ALL, install the PreSonus driver, reboot and then retry. If a device/driver is added after you open Jamulus, it might not be detected properly.

**Reply by henkdegroot** [reply]
@jedlevine , did you install the correct driver for the Studio 24C? There are 2 in the list which you can select during the install and the Studio USB driver is the you need to use. 
Also make sure to select the correct input from the 24C, as there are a few listed (even virtual inputs).

**Comment by dakhubgit** [comment]
Your keyboard mixer strip is up and you don't have the "Mute" button pressed?
